cucumber/api/features/step_definitions/request_steps.rb

```diff
@@ -103,6 +103,14 @@
   end
 end
 
+When(/^I authenticate Alice with Accept-Encoding header "([^"]*)"(?: with plain text body "([^"]*)")?$/) do |header, body|
```

Comment:
can we use the existing step for this? looks like you can add there an optional header

Reply:
I'm doing manipulations over the body and using `try_request false` to test the `401`case. Think that it's an overhead. Especially when we have `with parameters`, `with body`, `with username`, `with no Content-Header

Reply:
i will not block this PR for this but still think it can be generalized